src/Query/Wildcard.php

```diff
@@ -23,13 +23,14 @@ class Wildcard extends AbstractQuery
      */
     private $field;
 
-    public function __construct(string $field, string $value, float $boost = 1.0)
+    public function __construct(string $field, string $value, float $boost = 1.0, bool $caseInsensitive = true)
```

Comment:
From the ES documentation:
- the parameter is optional, thus "nullable" IMHO → in that case it should not appear in the query at all
- the default value is "false", not "true"

Would it make sense to set  the parameter only if a value is passed, and have it null by default? wdyt?

Reply:
why not, I'm changing it.